### PR TITLE
detailed result of show index

### PR DESCRIPTION
**Pull request by bright-starry-sky**
issue : https://github.com/vesoft-inc/nebula-graph/issues/1117

Current result of show indexes : 
```
(root@nebula) [test_show_space]> show tag indexes
+------------+--------+----------+
| Index Name | By Tag | Columns  |
+------------+--------+----------+
| "i1"       | "t1"   | "c1"     |
+------------+--------+----------+
| "i2"       | "t1"   | "c1, c2" |
+------------+--------+----------+
Got 2 rows (time spent 1263/1576 us)

Thu, 24 Jun 2021 13:36:47 CST

(root@nebula) [test_show_space]> show tag indexes by t1
+------------+----------+
| Index Name | Columns  |
+------------+----------+
| "i1"       | "c1"     |
+------------+----------+
| "i2"       | "c1, c2" |
+------------+----------+
Got 2 rows (time spent 1072/1392 us)

Thu, 24 Jun 2021 13:36:55 CST

(root@nebula) [test_show_space]> show edge indexes
+------------+---------+----------+
| Index Name | By Edge | Columns  |
+------------+---------+----------+
| "i3"       | "e1"    | "c1"     |
+------------+---------+----------+
| "i4"       | "e1"    | "c1, c2" |
+------------+---------+----------+
Got 2 rows (time spent 1075/1357 us)

Thu, 24 Jun 2021 13:37:03 CST

(root@nebula) [test_show_space]> show edge indexes by e1
+------------+----------+
| Index Name | Columns  |
+------------+----------+
| "i3"       | "c1"     |
+------------+----------+
| "i4"       | "c1, c2" |
+------------+----------+
Got 2 rows (time spent 1045/1318 us)
```

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

File: tests/tck/features/index/Index.IntVid.feature [hunk 1]
```diff
@@ -125,9 +125,9 @@ Feature: IndexTest_Vid_Int
       Show TAG INDEXES;
       """
     Then the result should be, in any order:
-      | Names                |
-      | "disorder_tag_index" |
-      | "single_tag_index"   |
+      | Index Name           | By Tag  | Columns      |
+      | "disorder_tag_index" | "tag_1" | "col3, col2" |
```
Comment (Shylock-Hg) [inline tests/tck/features/index/Index.IntVid.feature:129]: I think you could return columns as list type.

Comment (laura-ding) [reply tests/tck/features/index/Index.IntVid.feature:129]: agree

Comment (bright-starry-sky) [reply tests/tck/features/index/Index.IntVid.feature:129]: > agree, it can help users to get column name

For example?

Comment (laura-ding) [reply tests/tck/features/index/Index.IntVid.feature:129]: user can use it to check if the index columns are expected, the list is easier to handle than string.

Comment (bright-starry-sky) [reply tests/tck/features/index/Index.IntVid.feature:129]: > user can use it to check if the index columns are expected, the list is easier to handle than string.

Is it just an output format opinion? I think that's enough for the moment, it should be the same as `SHOW CREATE ...`.
In addition, if the user needs to determine whether an index is valid or not, this needs to be combined with the execution plan.

**Review by laura-ding — APPROVED** [review]

**Comment by bright-starry-sky** [comment]
Addressed @Shylock-Hg and @laura-ding 's comments. re-format the result from string to list.

**Review by yixinglu — DISMISSED** [review]

**Review by CPWstatic — APPROVED** [review]

File: src/parser/parser.yy [hunk 2]
```diff
@@ -2903,10 +2903,16 @@ show_sentence
          $$ = new ShowEdgesSentence();
     }
     | KW_SHOW KW_TAG KW_INDEXES {
-         $$ = new ShowTagIndexesSentence();
+         $$ = new ShowTagIndexesSentence(new std::string(""));
+    }
+    | KW_SHOW KW_TAG KW_INDEXES KW_BY name_label {
```
Comment (Shylock-Hg) [inline src/parser/parser.yy:2908]: What's the meaning of the `by label` ? And you should add case in tck with this.

**Comment by bright-starry-sky** [comment]
Corrected format of tck case . added more test case.

**Review by yixinglu — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Pull request merged**